setup.py

```diff
@@ -32,7 +32,7 @@
 NUMPY_DEP = 'numpy>=1.7'
 
 # these are required to use h5py
-RUN_REQUIRES = [NUMPY_DEP,]
+RUN_REQUIRES = [NUMPY_DEP, "cached-property"]
```

Comment:
Does this interact nicely with py3.8?

Reply:
No idea! Is there something you're aware of that might break it?